docs/src/navigations.md

```diff
@@ -222,6 +270,14 @@ await Promise.all([
 ]);
 ```
 
+```java
+// Using waitForNavigation with a callback prevents a race condition
+// between clicking and waiting for a navigation.
+page.waitForNavigation(() -> { // Waits for the next navigation
+  page.click("a"); // Triggers a navigation after a timeout
```

Comment:
remove "aftter a timeout"

Reply:
it'd make the example useless as waitForNavigation would be redundant. I'll update the snippet separately for all langs